sdk/PrebidMobile/PBTargetingParams.m

```diff
@@ -158,4 +159,29 @@ - (void)setItunesID:(NSString *)itunesID {
     _itunesID = itunesID;
 }
 
+-(void) setSubjectToGDPR:(BOOL)subjectToGDPR{
+    [[NSUserDefaults standardUserDefaults] setBool:subjectToGDPR forKey:PB_GDPR_SubjectToConsent];
+}
+
+-(void) setGdprConsentString:(NSString *)gdprConsentString{
+    
+    [[NSUserDefaults standardUserDefaults] setObject:gdprConsentString forKey:PB_GDPR_ConsentString];
+}
+
+-(BOOL) subjectToGDPR {
+    BOOL savedGDPR = YES;
```

Comment:
should be able to return nil case, which indicates publisher didn't set any GDPR.

Reply:
If nil & if no values are passed in the request, the impbus treats as subjectToGDPR as `YES`... so its ok to pass in YES by default...

Reply:
From OPENRTB Spec
The “Regs” object will signal whether or not the request is subject to GDPR regulations. It will do so via
the extension attribute “gdpr” which is an optional integer that indicates: 0 = No, 1 = Yes. Under
OpenRTB conventions for optional attributes, omission indicates Unknown.

0, 1 and unknown actually means different things to exchanges and buyers. For example, impbus will use a different way to process unknown situation.